OpenSim/Simulation/Wrap/WrapCylinder.cpp

```diff
@@ -76,7 +76,7 @@ WrapCylinder::~WrapCylinder()
 */
 void WrapCylinder::constructProperties()
 {
-    constructProperty_radius(-1.0);
```

Comment:
This change is backwards-incompatible.

Reply:
@aymanhab what do you prefer here? Maintaining backwards compatibility or having default property values that work (not forcing users to specify the value of this property)?

Reply:
@chrisdembia Thanks for bringing this up. Let's clarify who would be affected and how so we can see what makes sense:
1. Any valid model that comes from .osim file would have these attributes populated so default values are not in play.
2. Some old models that used to blow up because attributes were invalid will not blow up.
3. If you create a fresh object using the new default constructor and add it to the model (which is the recommended way in 4.0 since XML editing is more complicated/error-prone) then you can edit the model in the GUI without blowing up.
I'd lean to drop the blowup in 2 in favor of a meaningful use in 3 even if this technically is not backward compatible, but I sure would like to hear your thoughts.

Reply:
Ah okay yeah let's go with `1`